packages/eas-cli/src/commands/device/delete.ts

```diff
@@ -0,0 +1,125 @@
+import { getConfig } from '@expo/config';
+import { Flags } from '@oclif/core';
+import assert from 'assert';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import {
+  chooseSingleDeviceToDeleteAsync,
+  formatDeviceLabel,
+} from '../../credentials/ios/actions/DeviceUtils';
+import { AppleDeviceMutation } from '../../credentials/ios/api/graphql/mutations/AppleDeviceMutation';
+import { AppleDeviceQuery } from '../../credentials/ios/api/graphql/queries/AppleDeviceQuery';
+import { AppleTeamQuery } from '../../credentials/ios/api/graphql/queries/AppleTeamQuery';
+import formatDevice from '../../devices/utils/formatDevice';
+import Log from '../../log';
+import { Ora, ora } from '../../ora';
+import { findProjectRootAsync, getProjectAccountNameAsync } from '../../project/projectUtils';
+import { promptAsync, toggleConfirmAsync } from '../../prompts';
+
+export default class DeviceDelete extends EasCommand {
+  static description = 'remove a registered device from your account';
+
+  static flags = {
+    'apple-team-id': Flags.string(),
+  };
+
+  async runAsync(): Promise<void> {
+    let appleTeamIdentifier = (await this.parse(DeviceDelete)).flags['apple-team-id'];
+
+    const projectDir = await findProjectRootAsync();
+    const { exp } = getConfig(projectDir, { skipSDKVersionRequirement: true });
+    const accountName = await getProjectAccountNameAsync(exp);
+
+    let spinner: Ora;
+
+    if (!appleTeamIdentifier) {
+      spinner = ora().start('Fetching the list of teams for the project…');
+
+      try {
+        const teams = await AppleTeamQuery.getAllForAccountAsync(accountName);
+
+        if (teams.length) {
```

Comment:
```suggestion
        if (teams.length > 0) {
```
- It's always better to be explicit instead of depending on implicit conversion.
- Yes, I know this is copy-pasted.

Reply:
I'll adjust it in the original location too.